app/lib/utils/i18n.js

```diff
@@ -93,30 +93,172 @@ export const locales = [
   'zh-TW'
 ]
 
-// Defaine list of currencies that we will support.
+// Define list of currencies that we will support.
 export const currencies = [
-  'USD',
-  'EUR',
-  'JPY',
-  'GBP',
-  'CAD',
-  'KRW',
+  'AED',
```

Comment:
If we are going to define a complete list like this, could we pull the list from the `country-data-lookup` package instead, which is already installed and being used in this library?

See https://github.com/Zooz/country-data

You can fetch a list of all currencies buy calling `currencies.all`

Reply:
Sure can do. I'll take a look. This is just a list of all the currencies that Coinbase `https://api.coinbase.com/v2/exchange-rates?currency=btc` spits out.